internal/common/resource/resource.go

```diff
@@ -50,6 +58,8 @@ func (a ComputeResources) String() string {
 	return str
 }
 
+// Add function adds the provided values in map b to its corresponding existing
```

Comment:
I don't think every function needs to be commented.  I would look at some and see if they are not self explanatory.  This one seems probably obvious me by reading the function.